src/librustc/session/config.rs

```diff
@@ -1172,7 +1172,7 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
           "emit noalias metadata for mutable references"),
     dump_mir: Option<String> = (None, parse_opt_string, [UNTRACKED],
           "dump MIR state at various points in translation"),
-    dump_mir_dir: Option<String> = (None, parse_opt_string, [UNTRACKED],
+    dump_mir_dir: Option<String> = (Some(String::from("mir_dump")), parse_opt_string, [UNTRACKED],
```

Comment:
Could this be a `String` instead of `Option<String>` then?

Reply:
That's a good point! It's now non-optional.